gazelle/internal/swift/rules_from_srcs.go

```diff
@@ -23,16 +24,17 @@ func RulesFromSrcs(
 	var rules []*rule.Rule
 	switch moduleType {
 	case LibraryModuleType:
-		rules = rulesForLibraryModule(defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
+		rules = rulesForLibraryModule(defaultName, defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
 	case BinaryModuleType:
-		rules = rulesForBinaryModule(defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
+		rules = rulesForBinaryModule(defaultName, defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
 	case TestModuleType:
-		rules = rulesForTestModule(defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
+		rules = rulesForTestModule(defaultName, defaultModuleName, srcs, swiftImports, shouldSetVis, args.File)
 	}
 	return rules
 }
 
 var guiModules = mapset.NewSet("AppKit", "UIKit", "SwiftUI")
+var testModules = mapset.NewSet("XCTest", "XCUITest")
```

Comment:
Is this being used?

Reply:
Oh no I must have forgotten to clean this up. I can remove it now.